ui/app/AppLayouts/Wallet/popups/swap/SwapModalAdaptor.qml

```diff
@@ -67,6 +102,39 @@ QObject {
         return null
     }
 
+    function fetchSuggestedRoutes() {
+        // Identify new swap with a different uuid
+        d.uuid = Utils.uuid()
```

Comment:
What if the previous fetch hasn't finished yet?

Reply:
That logic is supposed to be covered by the issues klisted in https://github.com/status-im/status-desktop/pull/15046#discussion_r1628579098